src/backend/main.ts

```diff
@@ -102,6 +102,38 @@ export async function connect(config: connections.Config) {
   toast.success('Connected to forwarder successfully!')
 }
 
+// Adam Chen snapshot 1.0 helper functions
+
+function numberToBytes(number: number) {
```

Comment:
I suggest you use NDNts' existing converter: https://github.com/yoursunny/NDNts/blob/main/pkg/tlv/src/nni.ts

Reply:
oh. will do that.